helm/superset/values.yaml

```diff
@@ -27,6 +27,11 @@ nameOverride: ~
 # -- Provide a name to override the full names of resources
 fullnameOverride: ~
 
+# -- Global values for the Superset chart
+global:
+  # -- Labels to be added to all resources
+  extraLabels: {}
+
```

Comment:
Based on the comment here, these would be added to `all` resources. However, it seems we're not adding these to the pod templates (see for instance `.Values.supersetWorker.podLabels` - is this intentional?

Reply:
sorry, I just missed them. I'm adding it now